samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -557,20 +600,131 @@ CONFIG_AT_CMD_REQUEST_RESPONSE_BUFFER_LENGTH - Length of AT command request resp
    Modem responses longer than this length will be replaced with an error code message (-NRF_E2BIG).
    Cannot be less than 40 bytes.
 
+When using CoAP, the following additional configuration options are available:
+
+.. _CONFIG_COAP_SHADOW_CHECK_RATE_SECONDS:
+
+CONFIG_COAP_SHADOW_CHECK_RATE_SECONDS - Rate to check for shadow changes
+   How many seconds between requests for any change (delta) in the device shadow.
+
+.. _CONFIG_COAP_SHADOW_THREAD_STACK_SIZE:
+
+CONFIG_COAP_SHADOW_THREAD_STACK_SIZE - CoAP Shadow Thread Stack Size (bytes)
+   Sets the stack size (in bytes) for the shadow delta checking thread of the sample.
+
+.. _CONFIG_NRF_CLOUD_COAP_FOTA:
+
+CONFIG_NRF_CLOUD_COAP_FOTA - Enable FOTA with CoAP
+   The sample periodically checks for pending FOTA jobs.
+   The sample performs the FOTA update when received.
+
+If :kconfig:option:`CONFIG_NRF_CLOUD_COAP_FOTA` is enabled, these options additional are available:
+
+.. _CONFIG_COAP_FOTA_DL_TIMEOUT_MIN:
+
+CONFIG_COAP_FOTA_DL_TIMEOUT_MIN - CoAP FOTA download timeout
+    The time in minutes allotted for a FOTA download to complete.
+
+.. _CONFIG_COAP_FOTA_USE_NRF_CLOUD_SETTINGS_AREA:
+
+CONFIG_COAP_FOTA_USE_NRF_CLOUD_SETTINGS_AREA - Make FOTA compatible with other samples
+   Use the same settings area as the nRF Cloud FOTA library.
+
+.. _CONFIG_COAP_FOTA_SETTINGS_NAME:
+
+CONFIG_COAP_FOTA_SETTINGS_NAME - Settings identifier
+   Set the identifier for the CoAP FOTA storage if :kconfig:option:`CONFIG_COAP_FOTA_USE_NRF_CLOUD_SETTINGS_AREA` is not enabled.
+
+.. _CONFIG_COAP_FOTA_SETTINGS_KEY_PENDING_JOB:
+
+CONFIG_COAP_FOTA_SETTINGS_KEY_PENDING_JOB - Settings item key
+   Set the settings item key for pending FOTA job info if :kconfig:option:`CONFIG_COAP_FOTA_USE_NRF_CLOUD_SETTINGS_AREA` is not enabled.
+
+.. _CONFIG_COAP_FOTA_JOB_CHECK_RATE_MINUTES:
+
+CONFIG_COAP_FOTA_JOB_CHECK_RATE_MINUTES - FOTA job check interval (minutes)
+   How many minutes between requests for a FOTA job.
+
+.. _CONFIG_COAP_FOTA_THREAD_STACK_SIZE:
+
+CONFIG_COAP_FOTA_THREAD_STACK_SIZE - CoAP FOTA Thread Stack Size (bytes)
+   Sets the stack size (in bytes) for the FOTA job checking thread of the sample.
+
 .. include:: /libraries/modem/nrf_modem_lib/nrf_modem_lib_trace.rst
    :start-after: modem_lib_sending_traces_UART_start
    :end-before: modem_lib_sending_traces_UART_end
 
-.. _nrf_cloud_mqtt_multi_service_building_and_running:
+.. _nrf_cloud_multi_service_building_and_running:
 
 Building and running
 ********************
 
-.. |sample path| replace:: :file:`samples/cellular/nrf_cloud_mqtt_multi_service`
+.. |sample path| replace:: :file:`samples/cellular/nrf_cloud_multi_service`
 
 .. include:: /includes/build_and_run_ns.txt
 
-.. _nrf_cloud_mqtt_multi_service_building_wifi:
+.. _nrf_cloud_multi_service_coap_provisioning:
+
+Provisioning a CoAP device on nRF Cloud
+***************************************
+
+For more information, see `nRF Cloud Utilities documentation`_.
```

Comment:
This might have a better place at the end of this section.

Reply:
I specifically chose this location to emphasize that this provisioning must occur.  I do not want users to follow the "Building With CoAP Support" directions without properly provisioning the device.

Reply:
Okay, but you might emphasize this a bit more, maybe saying something like: Read this first.

Reply:
I realized after viewing the built docs that the indent level of the Provisioning a CoAP device on nRF Cloud section was the same as the Building and running section.  I have swapped the two and it looks better to me now.

QA discovered a lack of documentation on a particular provisioning topic, so I added a 3 more sentences to the Provisioning a CoAP device on nRF Cloud.  If it needs changes, I can do a follow on PR.